server/src/router_utils.rs

```diff
@@ -0,0 +1,199 @@
+use once_cell::sync::OnceCell;
```

Comment:
I expect all of this to be in lib-router, will create an issue for the future migration